azavea/raster-foundry
# Revert `toggle` component and add `toggle-ow` component

## Overview

This PR is the sequel to #1871, in which our protagonist, with all that he experienced in the first installment, returns to set things right.

Namely, this PR reverts the changes to the `toggle` component and leaves it's two-way binding in tact. To address the issue that #1871 attempted to resolve, this PR adds a `toggle-ow` component (toggle one-way) that only handles one-way bindings. This allows us to provide a method call as a value to the toggle, which causes problems when the value is bound using a two-way binding (as methods aren't assignable).

This PR also changes parts of the application that were using a method to set the value of a toggle to use the new `toggle-ow` component.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

Optional. Screenshots, `curl` examples, etc.

##Notes

We should probably use the `toggle-ow` component when creating new parts of the application.

## Testing Instructions

 * Test the color-mode selection (this is using the new one-way toggle)
 * Test the color-correction toggles (this is using the original toggle)
 * Everything should work as expected

Closes #1900 
Closes #1901 
Closes #1905
